cloud/gc_dns.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python
```

Comment:
http://docs.ansible.com/ansible/faq.html#how-do-i-handle-python-pathing-not-having-a-python-2-x-in-usr-bin-python-on-a-remote-machine